lms/djangoapps/badges/utils.py

```diff
@@ -0,0 +1,12 @@
+"""
+Utility functions used by the badging app.
+"""
+from django.conf import settings
+
+
+def site_prefix():
```

Comment:
@nedbat Not as a convenient function. These statements are run individually in any part of the code that needs them. I wasn't sure where to put this function to replace those other instances.